src/Compilers/CSharp/Portable/Symbols/Source/SourceComplexParameterSymbol.cs

```diff
@@ -262,6 +262,16 @@ private ConstantValue DefaultSyntaxValue
                         }
                     }
 
+                    // Check availability of `DecimalConstantAttribute`.
```

Comment:
>/ Check availability of `DecimalConstantAttribute`. [](http://example.com/codeflow?start=21&length=51)

Consider incorporating this check into the previous ```if```, under ```if (!_lazyDefaultSyntaxValue.IsBad)```. Also, we probably don't want to perform this check when there is a constant value from attribute (see SourceParameterSymbolBase.AddSynthesizedAttributes not trying to synthesize the attribute in this case). There could be a situation when we cannot find the well-known member due to an ambiguity, but the explicit attribute application is successful, we shouldn't be complaining in a case like that. Let's try to create a test that exhibits this behavior with the current implementation.

Reply:
Using MakeTypeMissing test helper should be enough for the test.

Reply:
This thread was marked as resolved, but is not addressed. I think it is still relevant. Reactivating.

Reply:
You are right, it wasn't resolved. Now it is. The code is inside the `if` after moving the other part elsewhere.